src/Apps/ArtQuiz/__tests__/ArtQuizResults.jest.tsx

```diff
@@ -0,0 +1,53 @@
+import { ArtQuizResults } from "Apps/ArtQuiz/ArtQuizResults"
+import { render, screen } from "@testing-library/react"
+
+const translationFileMock = {
+  artQuizPage: {
+    results: {
+      emailButton: "Email My Results",
+      title: "Explore Your Quiz Results",
+      subtitle:
+        "Explore these collections and artists recommended for you based on your saved works. Follow them to see their latest works on your Artsy home.",
+      tabs: {
+        worksYouLiked: "Works You Liked",
+        recommendedCollections: "Recommended Collections",
+        recommendedArtists: "Recommended Artists",
+      },
+    },
+  },
+}
+
+jest.mock("react-i18next", () => {
+  return {
+    useTranslation: () => ({
+      t: (dotNotation: string) => {
+        const [page, ...nestedKeys] = dotNotation.split(".")
+
+        let temp: Record<string, any> = { ...translationFileMock[page] }
+
+        nestedKeys.forEach(key => {
+          temp = temp[key]
+        })
+
+        return (temp as unknown) as string
+      },
+    }),
+  }
+})
+
+describe("ArtQuizResults", () => {
+  it.each([
+    ["title", "Explore Your Quiz Results"],
+    [
+      "subtitle",
+      "Explore these collections and artists recommended for you based on your saved works. Follow them to see their latest works on your Artsy home.",
+    ],
+    ["email button", "Email My Results"],
+    ["tab", "Works You Liked"],
+    ["tab", "Recommended Collections"],
+    ["tab", "Recommended Artists"],
+  ])("displays the expected %s text", (_key, text) => {
```

Comment:
This seems... excessive for a simple render spec.

Reply:
Why?

Reply:
It's just like: Did the thing render correctly, y or n? A single spec will tell you that and is easier to read and maintain vs a dynamically constructed set of specs for every significant piece of text on the page.

Reply:
`blocking` The `NavBar` used to have a similar spec. It would break constantly and is a pain to maintain. It makes it difficult to search for the relevant spec when you see it in your test output. There's no benefit here, just a simple: did a single string render is fine. And in fact nothing of consequence is even being tested because the strings it's looking for are mocked.